src/panic-handler.md

```diff
@@ -0,0 +1,65 @@
+## #[panic_handler]
+
+`#[panic_handler]` is used to define the behavior of `panic!` in `#![no_std]` applications.
+The `#[panic_handler]` attribute must be applied to a function with signature `fn(&PanicInfo)
+-> !` and such function must appear *once* in the dependency graph of a binary / dylib / cdylib
+crate. The API of `PanicInfo` can be found in the [API docs].
+
+[API docs]: https://doc.rust-lang.org/nightly/core/panic/struct.PanicInfo.html
```

Comment:
This should be a relative link so it works offline

    ../core/panic/struct.PanicInfo.html

Reply:
Fixed!